### PR TITLE
Features/pagination logic and tiles corrections

### PR DESCRIPTION
**Pull request by marcin10lw**
For now we get our Pagination state from search parameters. currentPage state in movieSlice and peopleSlice are redundant for now, but we'll know more when we setup query searching 🙂

### CODE REVIEW

File: src/common/Pagination/index.js [hunk 1]
```diff
@@ -12,29 +14,34 @@ import {
 } from "./styled";
 
 const Pagination = () => {
-  const [currentPage, setCurrentPage] = useState(1);
+  const dispatch = useDispatch();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const currentPage = Number(searchParams.get("page")) || 1;
+  const totalPages = 500;
```
Comment (HelkaPL) [inline src/common/Pagination/index.js:18]: How about limit it for max 500 ( API max ) but still geting proper value if there were less ??

File: src/features/people/ActorPage/About/index.js [hunk 2]
```diff
@@ -22,11 +23,13 @@ const About = () => {
         <Name>{actorInfo.name}</Name>
         <BirthInfo>
           <div>
-            <SpecificInfo>Date of birth:</SpecificInfo>
-            {actorInfo.birthday}
+            <SpecificInfo>
+              <OptionalInfo>date of</OptionalInfo> birth:
```
Comment (HelkaPL) [inline src/features/people/ActorPage/About/index.js:27]: ` </OptionalInfo> birth: ` is that `birth:` out for a reason ?

File: src/features/people/ActorPage/Crew/index.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { selectMoviesCrew } from "../actorSlice";
 
 const Crew = () => {
   const moviesCrew = useSelector(selectMoviesCrew);
-
+  console.log(moviesCrew);
```
Comment (HelkaPL) [inline src/features/people/ActorPage/Crew/index.js]: I hope that's `conslole.log() ` only something you just forgot to remove ?

**Review by HelkaPL — APPROVED** [review]
I see a lot of good work here. 

Can't wait to merge it. Looks like it will be a shortcut for searchResults :)

File: src/features/people/ActorPage/About/index.js (hunk 2 above)
Comment (marcin10lw) [reply src/features/people/ActorPage/About/index.js:27]: In figma there is only birth on mobile view

File: src/common/Pagination/index.js (hunk 1 above)
Comment (marcin10lw) [reply src/common/Pagination/index.js:18]: What do you mean

Comment (HelkaPL) [reply src/common/Pagination/index.js:18]: It's now a fixed value, right?
But there will be some cases, when we will get an API response with less than 500 pages of results.
...
ok, mostly in search results, so we can update it with search abbility.

**Review by mikolajgraczyk — APPROVED** [review]
Well done 😎

**Pull request merged**